web/src/app/pages/create-survey/task-details/task-details.component.html

```diff
@@ -41,7 +42,7 @@ <h2>
 
   <div id="add-task-buttons-list">
     <add-task-button
-      *ngFor="let taskGroup of [TaskGroup.QUESTION, TaskGroup.PHOTO, TaskGroup.DROP_PIN, TaskGroup.DRAW_AREA, TaskGroup.SUGGEST_LOI]"
+      *ngFor="let taskGroup of [TaskGroup.QUESTION, TaskGroup.PHOTO, TaskGroup.DROP_PIN, TaskGroup.DRAW_AREA, TaskGroup.CAPTURE_LOCATION]"
```

Comment:
Can we move this list into the component to keep less logic here in the view template?